infra/bisector.py

```diff
@@ -58,6 +59,10 @@ class BuildData():
 
 def main():
   """Finds the commit SHA where an error was initally introduced."""
+  if os.getcwd() != os.path.dirname(
```

Comment:
We can make this easier to use by doing a chdir here instead of exiting.

Reply:
Done.